CHANGELOG.md

```diff
@@ -257,4 +257,22 @@ All notable changes to this project will be documented in this file. The format
  - Fix: AKO is not updating the ingress status when annotation `passthrough.ako.vmware.com/enabled: "true"` is added to the ingress.
  - Fixes LoadBalancer service creation with named ports in NodePortLocal deployment.
  - Fix: Every SEGroup used in the AviInfraSetting is getting configured with the labels even when `disableStaticRouteSync` is set to `true`.
- - Fix: AKO pod keeps getting error "panic: runtime error: slice bounds out of range" then goes into CrashLoopBackOff state.
\ No newline at end of file
+ - Fix: AKO pod keeps getting error "panic: runtime error: slice bounds out of range" then goes into CrashLoopBackOff state.
+
+## AKO-1.8.1
+
+### Added
+ - AKO now claims support for Kubernetes 1.24.
+ - Support for `AviInfraSetting` CRD in Passthrough Ingresses/Routes.
+ - Support for ISTIO mTLS authentication. (Tech-preview)
+ - Support for IPv4/IPv6 dual-stack networking. (Tech-preview)
+
+### Changed
+ - Added a knob to block the processing of objects in system namespaces in OpenShift and K8s clusters.
+ - For Insecure Ingress deployments, priority labels associated with pools will be case insensitive.
+ - In `NodePortLocal` deployment, AKO processes only the PODs with the annotation `nodeportlocal.antrea.io`.
```

Comment:
@swathinsankaran : AKO already do line number 273 and 274. But AKO used to do that in later in graph stage and in 1.8.1 we changed it in ingestion layer.  So we can present  these changes as an optimization or skip it.

Reply:
It will be better to remove this as it was already supported.